composer.json

```diff
@@ -12,19 +12,19 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.1|^8.0",
         "php-http/client-implementation": "^1.0",
         "php-http/discovery": "^1.2.1",
         "php-http/httplug": "^1.1 || ^2.0",
         "php-http/message": "^1.5",
-        "symfony/http-foundation": "~3.4|~4.3",
-        "symfony/serializer": "~3.4|~4.3"
+        "symfony/http-foundation": "^4.4|^5.1",
+        "symfony/serializer": "^4.4|^5.1"
     },
     "require-dev": {
         "guzzle/guzzle": "~3.7",
         "guzzlehttp/psr7": "^1.4",
         "php-http/mock-client": "^1.0",
-        "symfony/phpunit-bridge": "^3.4.19 || ^4.1.8"
+        "symfony/phpunit-bridge": "^4.4|^5.1"
```

Comment:
that one could be `^5.1`. the phpunit bridge 5.1 supports all the PHP versions you need.